adapters/tappx/tappx.go

```diff
@@ -51,14 +62,31 @@ func (a *TappxAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 			Message: "Error parsing bidderExt object",
 		}}
 	}
-
 	var tappxExt openrtb_ext.ExtImpTappx
 	if err := json.Unmarshal(bidderExt.Bidder, &tappxExt); err != nil {
 		return nil, []error{&errortypes.BadInput{
 			Message: "Error parsing tappxExt parameters",
 		}}
 	}
 
+	ext := Ext{
+		Bidder: Bidder{
+			Tappxkey: tappxExt.TappxKey,
+			Mktag:    tappxExt.Mktag,
+			Bcid:     tappxExt.Bcid,
+			Bcrid:    tappxExt.Bcrid,
+		},
+	}
+
+	jsonext, err := json.Marshal(ext)
+	if err == nil {
+		request.Ext = jsonext
+	} else {
+		return nil, []error{&errortypes.BadInput{
+			Message: "Error with tappxExt parameters",
+		}}
```

Comment:
This is an error which should never occur, but might sometimes if there is a memory corruption issue. I wouldn't classify this as BadInput. I recommend to just use a normal error which PBS-Core will classify as a fatal error and warn the host of a problem.

Something like:
```
return nil, []error{fmt.Errorf("Error writing TappX request.ext.")
```

Reply:
About change de docs. I open the pull request e88cf1c.
About the BadInput error I make the change in the e88cf1c commit.